ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
@samoht @avsm thx for clarification, I added this below.  I'm surprised by the sudden change of defaults between different travis flavours.